apps/web/src/app/file-explorer/file-explorer-table.tsx

```
@@ -167,48 +176,30 @@ export function FileExplorerTable({data, columns}: FileExplorerTableProps) {
                                             <div
-                                                className=" p-4 rounded-md border bg-card hover:bg-accent
-                                 transition-colors cursor-pointer flex flex-col items-center text-center"
-                                                onDoubleClick={() => {
-                                                    console.log(`Navigate to folder: ${item.name}`);
-                                                    // TODO: Implement folder navigation
-                                                }}
+                                                className="p-4 rounded-md border bg-card hover:bg-accent transition-colors cursor-pointer flex flex-col items-center text-center"
+                                                onDoubleClick={() => onFolderClick(item.id)}
+                                                role="button"
+                                                aria-label={`Open folder ${item.name}`}
                                             >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)